e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -62,3 +65,55 @@ const DATA_GRID_SELECTOR = '#container';
     await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR);
   });
 });
+
+test('Focused cell should not flick', async (t) => {
+  const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+  const firstCell = dataGrid.getDataCell(0, 0).element;
+  const secondCell = dataGrid.getDataCell(1, 0).element;
+  await t.click(firstCell);
+  await t.click(secondCell);
+
+  await ClientFunction(() => {
```

Comment:
Let's move this ```ClientFunction``` to named function on top of this test case or outside this test case to make test code more obvious and easy-to-read :)
It's hard to read the inline called ```ClientFunction```

E.g:
```typescript
 const initFocusCounter = (elementSelector: Selector) => ClientFunction(() => {
   const windowWithTestData = window as WindowWithTestData;
   windowWithTestData.counter = 0;
   
   // I'm not sure in this type case here.
   const element = elementSelector() as any as HTMLElement;
   element.addEventListener('focusin', () => {
     windowWithTestData.counter = windowWithTestData.counter + 1;
   });
 }, {
    dependencies: { element },
  });
```

And use this function in the test case code:
```typescript
test('...', async (t) => {
  // other lines
  await initFocusCounter(secondCell);
  // other lines
});
```